apps/e2e/cypress/e2e/FAPs.cy.ts

```diff
@@ -1130,6 +1131,186 @@ context('Fap reviews tests', () => {
         '1'
       );
     });
+
+    it('Officer should be able to update all un-assigned technical reviews to new contact', () => {
+      cy.assignProposalsToFaps({
+        fapInstruments: [
+          { instrumentId: newlyCreatedInstrumentId, fapId: createdFapId },
+        ],
+        proposalPks: [firstCreatedProposalPk],
+      });
+      cy.assignReviewersToFap({
+        fapId: createdFapId,
+        memberIds: [fapMembers.reviewer.id],
+      });
+      cy.addProposalTechnicalReview({
+        proposalPk: firstCreatedProposalPk,
+        status: TechnicalReviewStatus.FEASIBLE,
+        timeAllocation: firstProposalTimeAllocation,
+        submitted: false,
+        reviewerId: 0,
+        instrumentId: newlyCreatedInstrumentId,
+        comment: comment1,
+        publicComment: comment2,
+        questionaryId: initialDBData.technicalReview.questionaryId,
+      });
+      cy.createInstrument(instrument).then((result) => {
+        const createdInstrument = result.createInstrument;
+        if (createdInstrument) {
+          newlyCreatedInstrumentId = createdInstrument.id;
+        }
+      });
+      cy.assignProposalsToInstruments({
+        instrumentIds: [newlyCreatedInstrumentId],
+        proposalPks: [firstCreatedProposalPk],
+      });
+      const optionText = `${scientist2.firstName.slice(0, 3)} ${scientist2.lastName} (${scientist2.email})`;
+      cy.login('officer');
+      cy.visit(`/FapPage/${createdFapId}?tab=3`);
+
+      cy.finishedLoading();
+
+      cy.contains(proposal1.title)
+        .parent()
+        .find('[data-cy="view-proposal"]')
+        .click();
+
+      cy.finishedLoading();
+
+      cy.get('[role="dialog"]').contains('Technical review').click();
+
+      cy.contains(`${scientist2.firstName} ${scientist2.lastName}`).should(
+        'not.exist'
+      );
+
+      cy.visit(`/Instruments`);
+
+      cy.get('[aria-label="Edit"]').eq(1).click();
+
+      cy.get('[data-cy=instrument-contact-surname]')
+        .type(scientist2.lastName)
+        .get('[data-cy=findUser]')
+        .click();
```

Comment:
This search box only apears with the `USER_SEARCH_FILTER` is active (ie in stfc mode and not in e2e mode) can you make the test work with this feature off as well? An example here: https://github.com/UserOfficeProject/user-office-core/commit/e5019d9ee7e10bf8c6db3d62ec8470bf36bc72c3